data-raw/prisoners-point/prisoners-point.R

```diff
@@ -94,12 +94,12 @@ ppoint_model_results %>%
 
 # model input is air temp
 model_input <- stockton_air_monthly %>%
-  filter(between(year(date), 1979, 2000))
+  filter(between(year(date), 1980, 2000))
 
 prisoners_point_water_temps <-
   tibble(
     date = model_input$date,
-    water_temp = predict(rri_water_temp_model, newdata = model_input[, "air_temp"]),
+    water_temp = predict(ppoint_temp_model, newdata = model_input[, "air_temp"]),
```

Comment:
I had to change the model to `ppoint_temp_model`  which was generated above. The original model `rri_water_temp_model` does not exist in this script. (values remained the same from 1980-2000)